packages/app/src/components/Layout/DefaultLayout/DefaultAxis.js

```diff
@@ -111,16 +112,48 @@ class Axis extends React.Component {
                 onDrop={this.onDrop}
             >
                 <div style={styles.label}>{getAxisName(this.props.axisId)}</div>
-                <div style={styles.content}>
-                    {this.props.axis.map(dimensionId => (
-                        <Chip
-                            key={`${this.props.axisId}-${dimensionId}`}
-                            onClick={this.props.getOpenHandler(dimensionId)}
-                            axisId={this.props.axisId}
-                            dimensionId={dimensionId}
-                        />
-                    ))}
-                </div>
+                <Droppable
+                    droppableId={this.props.axisId}
+                    direction="horizontal"
+                >
+                    {provided => (
+                        <div
+                            style={styles.content}
+                            ref={provided.innerRef}
+                            {...provided.droppableProps}
+                        >
+                            {this.props.axis.map((dimensionId, index) => {
+                                const key = `${this.props.axisId}-${dimensionId}`
+
+                                return (
+                                    <Draggable
+                                        key={key}
+                                        draggableId={key}
+                                        index={index}
+                                    >
+                                        {provided => (
+                                            <div
+                                                ref={provided.innerRef}
+                                                {...provided.draggableProps}
+                                                {...provided.dragHandleProps}
+                                            >
+                                                <Chip
+                                                    key={key}
```

Comment:
is it ok to reuse the same key for a different element? append "-chip" or "-child"?

Reply:
I guess it's ok, but I will add the prefix/postfix.

Reply:
Chip shouldn’t need a key at all here, right?

Reply:
Right, it shouldn't be needed.